.github/workflows/pytest-gpu.yml

```diff
@@ -83,14 +82,26 @@ jobs:
               git fetch
               git checkout ${{ steps.extract_branch.outputs.branch }}
           fi
+          # Set error trap to catch any failing test
+          err=0
+          trap 'err=1' ERR
           # OpenMP offloading
-          DEVITO_ARCH=clang DEVITO_PLATFORM=nvidiaX py.test tests/test_gpu_openmp.py
-          DEVITO_ARCH=clang DEVITO_PLATFORM=nvidiaX py.test examples/seismic/elastic/elastic_example.py
-          DEVITO_ARCH=clang DEVITO_PLATFORM=nvidiaX py.test examples/seismic/viscoelastic/viscoelastic_example.py
+          mpicc --version
+          which mpicc
+          mpirun --version
+          mpiexec --version
```

Comment:
debug leftover?

Reply:
Sure yes. Intentionally left for some testing, will remove.